code/game/machinery/requests_console.dm

```diff
@@ -352,7 +352,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/requests_console, 30)
 	updateUsrDialog()
 
 /obj/machinery/requests_console/say_mod(input, list/message_mods = list())
-	if(spantext_char(input, "!", -3))
+	if(findtext(input, "!!!"))
```

Comment:
Why the "!!!"?

Reply:
I believe it's only catching the one string which ends !!!, the last did roughly the same but started looking 3 chars before the end

Reply:
That's not right. `spantext_char` returns the amount of matches for "the needle" it can find past the index.
If the index is negative, it starts from the end.

This is checking if any of the last three chars in input are !'s.